### PR TITLE
Creating project

### PR DESCRIPTION
**Pull request by Dziarnek**

### CODE REVIEW

File: js/scripts.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+};
+
+function moveLastSlide() {
+    var firstSlide = carouselList.find("li:first");
+    var lastSlide = carouselList.find("li:last");
+
+    firstSlide.before(lastSlide);
+};
+
+prev.click(function(){
```
Comment (matwjablonski) [inline js/scripts.js]: use method `on`

File: js/scripts.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    clearInterval(slidesInterval);
+    prevSlide();
+    slidesInterval = setInterval(changeSlide, 5000);
+});
+
+    //CONTROLS
+
+var controlsInterval = setInterval(checkPosition, 100);
+
+function checkPosition() {
```
Comment (matwjablonski) [inline js/scripts.js:51]: better make this function with checking index of element - in jQuery is method `.index()`

File: main.sass [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        li
+            display: inline-block
+            border-radius: 50px
+            border: 3px solid white
+            background-color: inherit
+            width: 20px
+            height: 20px
+
+
+
```
Comment (matwjablonski) [inline main.sass]: only one empty line on the end of document

**Review by matwjablonski — CHANGES_REQUESTED** [review]

File: js/scripts.js (hunk 2 above)
Comment (Dziarnek) [reply js/scripts.js:51]: I'm not sure how - index of each slide changes all the time and the only thing that doesn't change is id. That's why function is checking id insted of index.

Comment (matwjablonski) [reply js/scripts.js:51]: yes, but you can attach an index to e.g. `data-index=""` and operate on it, remember that your function should be more flexible and reusable ;)

**Review by matwjablonski — APPROVED** [review]

**Pull request merged**